### DISCUSSION TITLE
Welcome to imgkit Discussions!

### DISCUSSION THREAD

**Discussion by jarrekk** [jarrekk/imgkit#65] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gayu19** [comment]
Hey All, 
I have a query related to options how should we count 
"crop-h": "3",
"crop-w": "3"

What is the measuring parameter of "3" for image, is it pixel? or what?

**Comment by Rahul040202** [comment]
While generating image from html  , the css properties are not applied properly onto the html . How do I resolve this
